### DISCUSSION TITLE
TS0501B and do_not_disturb

### DISCUSSION THREAD

**Discussion by tequilaj-gh** [zigpy/zha-device-handlers#3292] [General]
I got this dimmable led controller (_TZ3210_f8dqbuze) that always starts in power on mode after a power failure. 
Zigbee2mqtt has support for a do_not_disturb, that when it is set to true, sets the light to not turn on after the outage.
Where can I find this option in zha?

**Comment by TheJulianJES** [comment]
If an option like that exists, you should have an entity. If not, create an issue requesting that feature to be added and fill-out the issue template.
